### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by payalm17** [zcq15/ACDNet#2]
What is the loss function formula used for training the model?
